core_v2/sources/domains.move

```diff
@@ -489,13 +488,15 @@ module aptos_names_v2::domains {
             );
         };
 
-        let reverse_lookup_result = get_reverse_lookup(account_addr);
-        if (option::is_none(&reverse_lookup_result)) {
-            // If the user has no reverse lookup set, set the user's reverse lookup.
-            set_reverse_lookup(sign, subdomain_name, domain_name);
```

Comment:
`set_reverse_lookup` internally calls `set_target_address` and `set_reverse_lookup_internal`